ocp_resources/node_network_configuration_policy.py

```diff
@@ -254,13 +257,20 @@ def _ports_backup(self, ip_family):
         for port in self.ports:
             _port = self._get_port_from_nns(port_name=port)
             if _port:
-                self.ipv4_ports_backup_dict[port] = _port[ip_family]
+                if ip_family == IPV4_STR:
+                    self.ipv4_ports_backup_dict[port] = _port[ip_family]
+                elif ip_family == IPV6_STR:
+                    self.ipv6_ports_backup_dict[port] = _port[ip_family]
+                else:
+                    raise TypeError(
```

Comment:
Should raise `ValueError`
(see https://docs.python.org/3/library/exceptions.html#TypeError)

Reply:
Ack.